### ISSUE TITLE
[BUG] ELK recovery failed while trying to restore the snapshot

### ISSUE THREAD

**Issue by przemyslavic** [epiphany-platform/epiphany#1380]
**Describe the bug**
ELK recovery process fails when trying to restore the snapshot.

**To Reproduce**
1. execute `epicli backup -b build/elkbackup/ -f elkbackup/backup_config.yml`
3. execute `epicli recovery -b build/elkbackup/ -f elkbackup/backup_config.yml`

**Expected behavior**
Elasticsearch snapshot was successfully restored.

**Log**

```12:00:18 INFO cli.engine.ansible.AnsibleCommand - fatal: [ec2-34-246-176-62.eu-west-1.compute.amazonaws.com]: FAILED! => {"changed": false, "content": "{\"error\":{\"root_cause\":[{\"type\":\"snapshot_restore_exception\",\"reason\":\"[epiphany:20200623-111653/a6OQSjloTOqjHrZ5MeYilw] cannot restore index [filebeat-6.8.5-2020.06.23] because an open index with same name already exists in the cluster. Either close or delete the existing index or restore the index under a different name by providing a rename pattern and replacement name\"}],\"type\":\"snapshot_restore_exception\",\"reason\":\"[epiphany:20200623-111653/a6OQSjloTOqjHrZ5MeYilw] cannot restore index [filebeat-6.8.5-2020.06.23] because an open index with same name already exists in the cluster. Either close or delete the existing index or restore the index under a different name by providing a rename pattern and replacement name\"},\"status\":500}", "content_length": "743", "content_type": "application/json; charset=UTF-8", "elapsed": 0, "json": {"error": {"reason": "[epiphany:20200623-111653/a6OQSjloTOqjHrZ5MeYilw] cannot restore index [filebeat-6.8.5-2020.06.23] because an open index with same name already exists in the cluster. Either close or delete the existing index or restore the index under a different name by providing a rename pattern and replacement name", "root_cause": [{"reason": "[epiphany:20200623-111653/a6OQSjloTOqjHrZ5MeYilw] cannot restore index [filebeat-6.8.5-2020.06.23] because an open index with same name already exists in the cluster. Either close or delete the existing index or restore the index under a different name by providing a rename pattern and replacement name", "type": "snapshot_restore_exception"}], "type": "snapshot_restore_exception"}, "status": 500}, "msg": "Status code was 500 and not [200]: HTTP Error 500: Internal Server Error", "redirected": false, "status": 500, "url": "https://10.1.3.243:9200/_snapshot/epiphany/20200623-111653/_restore"}```

As soon as all indexes are deleted, but before the snapshot is restored, filebeat creates an index, which causes the described error.

```

    - name: Delete all indices
      uri:
        <<: *uri
        url: "{{ elasticsearch_endpoint }}/_all"
        method: DELETE

    - name: Restore the snapshot
      uri:
        <<: *uri
        url: "{{ elasticsearch_endpoint }}/_snapshot/{{ elasticsearch_snapshot_repository_name }}/{{ snapshot_name }}/_restore"
        method: POST
```

**Comment by sk4zuzu** [comment]
From the first look, it seems to be a race condition between snapshot restore and kibana / filebeat processes. We delete all indices, but they are instantly recreated and cause conflicts.

**Comment by sk4zuzu** [comment]
It seems there are three ways to handle this:
1. WAF (web application firewall) blocking calls to "/index" endpoints (nginx could be used).
2. Iptables firewall blocking access  to specific ports, preventing kibana and filebeat (and possibly others) instances from connecting to elasticsearch.
3. Stop / restore / start cycle including all kibana and filebeat instances (seems to be the simplest :+1:).

**Comment by przemyslavic** [comment]
Fixed in #1381.
